currency/tasks.py

```diff
@@ -35,6 +27,30 @@ def _privat():
             }
             save_rate_data(mch.SR_PRIVAT, rate_kwargs)
 
+@shared_task()
+def _privat_4_years():
+    i = 1
+    days = 365*4
+    while i <= days:
+        current_date = datetime.now()-timedelta(days=1)
+        str_current_date = current_date.strftime('%d.%m.%Y')
+        url = f'https://api.privatbank.ua/p24api/exchange_rates?json&date={str_current_date}'
+        response = requests.get(url)
```

Comment:
https://requests.readthedocs.io/en/master/user/quickstart/#passing-parameters-in-urls